lib/workorder-status/workorder-status-directive.js

```diff
@@ -0,0 +1,42 @@
+var CONSTANTS = require('../constants');
+
+var getStatusIcon = function(status) {
+  var statusIcon;
+  switch (status) {
+  case 'In Progress':
+    statusIcon = 'autorenew';
+    break;
+  case 'Complete':
```

Comment:
Outside the scope of this JIRA. 

There is a JIRA for internationalisation of the modules, I'd say it will be done as part of that.